tools/build.sh

```diff
@@ -360,13 +360,31 @@ awk '
 # make redirect file available at /request/ too
 cp dist/index.html dist/request
 
+# Build RSA iframe
+output "🔑  Building RSA Iframe"
+cp src/lib/rsa/sandboxed/forge.min.js dist/lib/rsa/sandboxed/forge.min.HASH.js
+# Integirty check that forge.min.js is from node-forge 1.3.1
+RSA_IFRAME_FORGE_HASH="3Gf9EyQnrZbJZmyESzlWVBPEDdsfLQY8U1Evv204ff0="
+RSA_IFRAME_FORGE_NAME=$(add_hash_to_file_name dist/lib/rsa/sandboxed/forge.min.HASH.js)
+cp src/lib/rsa/sandboxed/RSAKeysIframe.js dist/lib/rsa/sandboxed/RSAKeysIframe.HASH.js
+RSA_IFRAME_SCRIPT_HASH=$(make_file_hash dist/lib/rsa/sandboxed/RSAKeysIframe.HASH.js)
+RSA_IFRAME_SCRIPT_NAME=$(add_hash_to_file_name dist/lib/rsa/sandboxed/RSAKeysIframe.HASH.js)
+cp src/lib/rsa/sandboxed/RSAKeysIframe.html dist/lib/rsa/sandboxed/
+# Note: requests in a sandboxed iframe are considered cross origin requests, for which, in addition to integrity checks,
+# also cors headers must be checked, see https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity.
```

Comment:
What headers must be sent with which values?

Reply:
https://github.com/nimiq/keyguard/pull/524#discussion_r2182339196